### ISSUE TITLE
toggle animations is not working

### ISSUE THREAD

**Issue by DennisAdelmann** [miguel-perez/smoothState.js#124]
Hey Guys,
i tried to embed the smoothState.js into my project. But for any reasons the 
"animation-direction: alternate-reverse;" is not working. The element is only fading in but not out.
Can someone help me? :) 

HTML

``` html
<div id="overview" class="m-scene">
<div class="next_project">
<a href="archive02.php"><img src="img/assets/arrow_right.svg"></a>
</div>
<!-- Classes that define elment animations -->
<div class="scene_element scene_element--fadein">
<div class="project_headline">
<a href="archive02.php">P01</a>
</div>
 </div>
</div>

…

        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.11.1.min.js"><\/script>')</script>
        <script src="jquery.smoothState.js"></script>
        <script src="js/plugins.js"></script>
        <script src="js/main.js"></script>

        <!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
        <script>
            (function(b,o,i,l,e,r){b.GoogleAnalyticsObject=l;b[l]||(b[l]=
            function(){(b[l].q=b[l].q||[]).push(arguments)});b[l].l=+new Date;
            e=o.createElement(i);r=o.getElementsByTagName(i)[0];
            e.src='//www.google-analytics.com/analytics.js';
            r.parentNode.insertBefore(e,r)}(window,document,'script','ga'));
            ga('create','UA-XXXXX-X');ga('send','pageview');
        </script>
        <script src="functions.js"></script>
    </body>
</html>

```

CSS

``` css
.m-scene .scene_element {
  animation-duration: 5.25s;
  -webkit-animation-duration: 5.25s;
  -moz-animation-duration: 5.25;
  transition-timing-function: ease-in;
  -webkit-transition-timing-function: ease-in;
  -moz-transition-timing-function: ease-in;
  animation-fill-mode: both;
  -webkit-animation-fill-mode: both;
  -moz-animation-fill-mode: both;
}

.m-scene .scene_element--fadein {
  animation-name: fadeIn;
  -webkit-animation-name: fadeIn;
  -moz-animation-name: fadeIn;
}

.m-scene.is-exiting .scene_element {
  animation-direction: alternate-reverse;
  -webkit-animation-direction: alternate-reverse;
  -moz-animation-direction: alternate-reverse;
}

/*
 * Keyframes
 */
@keyframes fadeIn {
  0% { opacity: 0; }
  100% { opacity: 1; }
}

@-webkit-keyframes fadeIn {
  0% { opacity: 0; }
  100% { opacity: 1; }
}

@-moz-keyframes fadeIn {
  0% { opacity: 0; }
  100% { opacity: 1; }
}
```

JS

``` js
;(function($) {
  'use strict';
  var $body = $('html, body'),
      content = $('#overview').smoothState({
        // Runs when a link has been activated
        onStart: {
          duration: 250, // Duration of our animation
          render: function (url, $container) {
            // toggleAnimationClass() is a public method
            // for restarting css animations with a class
            content.toggleAnimationClass('is-exiting');
            // Scroll user to the top
            $body.animate({scrollTop: 0});
          }
        }
      }).data('smoothState');
})(jQuery);
```

**Comment by miguel-perez** [comment]
Hi there,

I've realized it would best to move [personal support](https://github.com/miguel-perez/smoothState.js#need-help) requests over to Stack Overflow. There's a special [smoothstate.js](http://stackoverflow.com/tags/smoothstate.js) label for us there. This will help us keep the Github issue tracker clean for bugs, pull requests, and enhancements. If you're not having any luck on Stack Overflow, feel free to join the [Gitter room](https://gitter.im/miguel-perez/smoothState.js) and ping me there. I'll try to help as much as I can.

That said, I'll be closing out this issue.
